src/components/tabs/tabs.component.js

```diff
@@ -356,6 +358,8 @@ Tabs.propTypes = {
     "no right side",
     "no sides",
   ]),
+  /** sets tabsHeaderWidth to TabHeader */
+  tabsHeaderWidth: PropTypes.string,
```

Comment:
I think just call this prop `headerWidth`, no need to have `tabs` before it because it's a prop of the `Tabs` component

Reply:
I had in mind "we can change headers in every tab component" not the header tabs component itself :D  but you are right, it headerWidth is enough anyway